firstQuestion.js

```diff
@@ -0,0 +1,43 @@
+const set1= new Array(3,1,7,9);
+const set2= new Array(2,4,1,9,3);
+let Distinct= new Array();
```

Comment:
Use `const` here, the `Distinct` variable (as a reference) doesn't change in your code, thus no need to use `let`. Take a look at this [Stack Overflow discussion](https://stackoverflow.com/questions/46553062/what-is-the-preferred-declaration-convention-for-objects-or-arrays-const-or-let) for better understanding.

Also, in regards to the name, JavaScript uses _camelCase_ naming convention for naming variables and functions, not _PascalCase_ (`distinct` not `Distinct`).

And again same comment as above, use array literals for such cases, so the final output is

```js
const distinct = [];
```